cmd/kubeadm/app/phases/addons/dns/versions.go

```diff
@@ -38,3 +39,18 @@ func GetKubeDNSManifest(kubeVersion *version.Version) string {
 	// In the future when the kube-dns version is bumped at HEAD; add conditional logic to return the right manifest
 	return v170AndAboveKubeDNSDeployment
 }
+
+// GetCoreDNSVersion returns the right CoreDNS version for a specific k8s version
+func GetCoreDNSVersion(kubeVersion *version.Version) string {
```

Comment:
Let's use the same functions -- don't create a new one. Instead, choose what version to return based on DNS provider.
This way the right values will be shown for upgrades as well.